Cargo.toml

```diff
@@ -44,6 +44,7 @@ floem_reactive = { path = "reactive", version = "0.1.0" }
 floem-winit = { version = "0.29.4", features = ["rwh_05"] }
 image = { version = "0.24", features = ["jpeg", "png"] }
 copypasta = { version = "0.10.0", default-features = false, features = ["wayland", "x11"] }
+floem-window-vibrancy = "0.4.3"
```

Comment:
Why does it have to be separate crate?

Reply:
Also it would've been *nice* to ask us first about publishing crate with `floem` name